# To rebuild this patch, run
  if x=$(quilt top) && [ "$x" = resync_rules_base.diff ]; then
    echo Regenerating $x
  else
    echo 'You need to apply resync_rules_base.diff with "quilt push -f" first'
    exit 1
  fi
  ./debian/rules config.status
  cd rules/compat
  quilt add base.l[2-4]*.part base.m*.part
  rm -f base.l[2-4]*.part base.m*.part
  make
  cd ..
  quilt add base base.lst
  rm -f base base.lst
  make base base.lst
  cd ..
  quilt refresh --sort
  exit 0

Index: xkb-data/rules/compat/base.l2_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.l2_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.l2_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -27,7 +27,7 @@
   ben(basic)	=	+in(ben):2
   ben(probhat)	=	+in(ben_probhat):2
   dev(basic)	=	+in(deva):2
-  dvorak($dvoraklayouts)	=	+%l(dvorak):2
+  dvorak($dvoraklayouts)	=	+%v(dvorak):2
   dvorak(basic)	=	+us(dvorak):2
   dvorak(pl_basic)	=	+pl(dvorak):2
   dvorak(pl)	=	+pl(dvorak_quotes):2
Index: xkb-data/rules/compat/base.l2v2_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.l2v2_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.l2v2_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -1,7 +1,7 @@
   ben		basic	=	+in(ben):2
   ben		probhat	=	+in(ben_probhat):2
   dev		basic	=	+in(deva):2
-  dvorak		$dvoraklayouts	=	+%l(dvorak):2
+  dvorak		$dvoraklayouts	=	+%v(dvorak):2
   dvorak		basic	=	+us(dvorak):2
   dvorak		pl_basic	=	+pl(dvorak):2
   dvorak		pl	=	+pl(dvorak_quotes):2
Index: xkb-data/rules/compat/base.l3_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.l3_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.l3_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -27,7 +27,7 @@
   ben(basic)	=	+in(ben):3
   ben(probhat)	=	+in(ben_probhat):3
   dev(basic)	=	+in(deva):3
-  dvorak($dvoraklayouts)	=	+%l(dvorak):3
+  dvorak($dvoraklayouts)	=	+%v(dvorak):3
   dvorak(basic)	=	+us(dvorak):3
   dvorak(pl_basic)	=	+pl(dvorak):3
   dvorak(pl)	=	+pl(dvorak_quotes):3
Index: xkb-data/rules/compat/base.l3v3_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.l3v3_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.l3v3_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -1,7 +1,7 @@
   ben		basic	=	+in(ben):3
   ben		probhat	=	+in(ben_probhat):3
   dev		basic	=	+in(deva):3
-  dvorak		$dvoraklayouts	=	+%l(dvorak):3
+  dvorak		$dvoraklayouts	=	+%v(dvorak):3
   dvorak		basic	=	+us(dvorak):3
   dvorak		pl_basic	=	+pl(dvorak):3
   dvorak		pl	=	+pl(dvorak_quotes):3
Index: xkb-data/rules/compat/base.l4_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.l4_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.l4_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -27,7 +27,7 @@
   ben(basic)	=	+in(ben):4
   ben(probhat)	=	+in(ben_probhat):4
   dev(basic)	=	+in(deva):4
-  dvorak($dvoraklayouts)	=	+%l(dvorak):4
+  dvorak($dvoraklayouts)	=	+%v(dvorak):4
   dvorak(basic)	=	+us(dvorak):4
   dvorak(pl_basic)	=	+pl(dvorak):4
   dvorak(pl)	=	+pl(dvorak_quotes):4
Index: xkb-data/rules/compat/base.l4v4_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.l4v4_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.l4v4_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -1,7 +1,7 @@
   ben		basic	=	+in(ben):4
   ben		probhat	=	+in(ben_probhat):4
   dev		basic	=	+in(deva):4
-  dvorak		$dvoraklayouts	=	+%l(dvorak):4
+  dvorak		$dvoraklayouts	=	+%v(dvorak):4
   dvorak		basic	=	+us(dvorak):4
   dvorak		pl_basic	=	+pl(dvorak):4
   dvorak		pl	=	+pl(dvorak_quotes):4
Index: xkb-data/rules/compat/base.ml1_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.ml1_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.ml1_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -27,7 +27,7 @@
   *		ben(basic)			=	pc+in(ben)
   *		ben(probhat)			=	pc+in(ben_probhat)
   *		dev(basic)			=	pc+in(deva)
-  *		dvorak($dvoraklayouts)			=	pc+%l(dvorak)
+  *		dvorak($dvoraklayouts)			=	pc+%v(dvorak)
   *		dvorak(basic)			=	pc+us(dvorak)
   *		dvorak(pl_basic)			=	pc+pl(dvorak)
   *		dvorak(pl)			=	pc+pl(dvorak_quotes)
Index: xkb-data/rules/compat/base.ml1v1_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.ml1v1_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.ml1v1_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -1,7 +1,7 @@
   *		ben		basic		=	pc+in(ben)
   *		ben		probhat		=	pc+in(ben_probhat)
   *		dev		basic		=	pc+in(deva)
-  *		dvorak		$dvoraklayouts		=	pc+%l(dvorak)
+  *		dvorak		$dvoraklayouts		=	pc+%v(dvorak)
   *		dvorak		basic		=	pc+us(dvorak)
   *		dvorak		pl_basic		=	pc+pl(dvorak)
   *		dvorak		pl		=	pc+pl(dvorak_quotes)
Index: xkb-data/rules/compat/base.ml_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.ml_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.ml_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -27,7 +27,7 @@
   *		ben(basic)			=	pc+in(ben)
   *		ben(probhat)			=	pc+in(ben_probhat)
   *		dev(basic)			=	pc+in(deva)
-  *		dvorak($dvoraklayouts)			=	pc+%l(dvorak)
+  *		dvorak($dvoraklayouts)			=	pc+%v(dvorak)
   *		dvorak(basic)			=	pc+us(dvorak)
   *		dvorak(pl_basic)			=	pc+pl(dvorak)
   *		dvorak(pl)			=	pc+pl(dvorak_quotes)
Index: xkb-data/rules/compat/base.mlv_s.part
===================================================================
--- xkb-data.orig/rules/compat/base.mlv_s.part	2007-09-18 17:02:08.000000000 +0200
+++ xkb-data/rules/compat/base.mlv_s.part	2007-09-18 17:04:14.000000000 +0200
@@ -1,7 +1,7 @@
   *		ben		basic		=	pc+in(ben)
   *		ben		probhat		=	pc+in(ben_probhat)
   *		dev		basic		=	pc+in(deva)
-  *		dvorak		$dvoraklayouts		=	pc+%l(dvorak)
+  *		dvorak		$dvoraklayouts		=	pc+%v(dvorak)
   *		dvorak		basic		=	pc+us(dvorak)
   *		dvorak		pl_basic		=	pc+pl(dvorak)
   *		dvorak		pl		=	pc+pl(dvorak_quotes)
